src/signature.c

```diff
@@ -235,6 +244,26 @@ static EVP_PKEY *load_key_file(const gchar *keyfile, GError **error)
 	return res;
 }
 
+#if ENABLE_OPENSSL_PKCS11_PROVIDERS
```

Comment:
You can use `G_GNUC_UNUSED` to avoid the `#if` and always have it compile-checked.